src/Script.cpp

```diff
@@ -1056,5 +1056,28 @@ void Script::_registerGlobals() {
   luaL_register(_L, NULL, globalLibs);
   lua_pop(_L, 1);
 }
+
+int CursorLibSetHandCursor(lua_State *L) {
```

Comment:
It's weird to not put this and `CursorLibSetHandCursor` in CursorLib.h, but it solves a problem of cyclic dependencies. This function needs `DGCheckProxy` and `ProxyToImage` from Proxy.h, but Proxy.h includes CursorLib.h and removing that dependency is not straight forward because Script.cpp depends on both CursorLib.h and Proxy.h. So declaring these functions as extern in CursorLib.h and implementing them in Script.cpp was just the easiest solution I could think of.